.github/workflows/ai-runner-live-pipelines-docker.yaml

```diff
@@ -220,9 +222,11 @@ jobs:
         if: steps.check_build.outputs.should_build == 'true'
         with:
           context: "{{defaultContext}}:runner"
+          file: docker/Dockerfile.live-app-noop
```

Comment:
do you know why it's  `runner/docker/Dockerfile.live-app-noop` on `changed-files` step above?